src/azure-cli/azure/cli/command_modules/aro/custom.py

```diff
@@ -49,7 +53,7 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
 
     aad = AADManager(cmd.cli_ctx)
     if client_id is None:
-        app, client_secret = aad.create_application('aro-%s' % random_id)
+        app, client_secret = aad.create_application(cluster_resource_group or 'aro-' + random_id)
```

Comment:
Also wasn't sure about this `cluster_resource_group` here.  Should we remove this and leave this as-is?

Reply:
+1 on this change